src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable arrow-body-style */
```

Comment:
- [ ] [REQUIRED] There are too many eslint-disable. Kindly enable `eslint` and fix the errors manually as it checks the code standard and helps us write consistent code with good quality.